src/components/layout.css

```diff
@@ -528,3 +888,30 @@ main {
   top: 2000px;
   width: 3px;
 }
+
+/* TOC */
+details,
+.toc {
+  cursor: pointer;
+}
+.toc h6 {
+  display: inline-block;
+  margin: 0;
+}
+.toc a {
+  font-family: var(--sans-serif);
+  text-decoration: none;
+  border-bottom: 1px dotted var(--gray3);
+}
+
+.toc li p {
+  margin: 0;
+}
+
+/* 3rd Party plugin styles. */
+/* Styles for gatsby-remark-autolink-headers plugin*/
+
+.autolink-headers {
+  position: absolute;
+  left: -4px;
```

Comment:
We probably wanna retain the orange fill in the link icon. It is more consistent with the previous design.

Without fill:
![image](https://user-images.githubusercontent.com/2608321/54109122-c871c280-4403-11e9-9983-ed0353c59799.png)

With orange fill:
![image](https://user-images.githubusercontent.com/2608321/54109161-dde6ec80-4403-11e9-82b8-d68915c4354c.png)


```suggestion
  left: -4px;
  fill: var(--orange3);
```

Reply:
Done in 0c63ac2